pysatModels/models/ucar_tiegcm.py

```diff
@@ -32,17 +40,19 @@
 
 platform = 'ucar'
 name = 'tiegcm'
-tags = {'': 'UCAR TIE-GCM file'}
-inst_ids = {'': ['']}
+tags = {'': 'UCAR TIE-GCM file', 'icon': 'UCAR TIE-GCM file produced for ICON.'}
+inst_ids = {'': list(tags.keys())}
 
-# specify using xarray (not using pandas)
+# Specify using xarray (not using pandas)
 pandas_format = False
 
 # ----------------------------------------------------------------------------
 # Instrument test attributes
 
-_test_dates = {'': {'': dt.datetime(2019, 1, 1)}}
-_test_download = {'': {'': False}}
+_test_dates = {'': {'': dt.datetime(2019, 1, 1),
+                    'icon': dt.datetime(2020, 1, 10)}}
+_test_download = {'': {'': False, 'icon': True}}
+_test_download_ci = {'': {'': False, 'icon': True}}
```

Comment:
```suggestion
_test_download = {'': {'': False, 'icon': True}}
```
Based on @aburrell 's comments, I recommend this update instead of the other one